contrib/coreos/user-data.example

```diff
@@ -1,161 +1,101 @@
-        --volume=/usr/share/ca-certificates:/etc/ssl/certs:ro \
-        -p 4001:4001 -p 2380:2380 -p 2379:2379 -p 7001:7001 \
-        --name ${ETCD_NAME} \
-        ${ETCD_IMAGE} \
-        -name ${ETCD_NAME} \
-        -data-dir /var/lib/etcd2 \
-        -advertise-client-urls http://${COREOS_PRIVATE_IPV4}:2379,http://${COREOS_PRIVATE_IPV4}:4001 \
-        -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 \
-        -initial-advertise-peer-urls http://${COREOS_PRIVATE_IPV4}:2380,http://${COREOS_PRIVATE_IPV4}:7001 \
-        -listen-peer-urls http://0.0.0.0:2380,http://0.0.0.0:7001 \
-        --heartbeat-interval ${ETCD_HEARTBEAT_INTERVAL} \
-        --election-timeout ${ETCD_ELECTION_TIMEOUT} \
-        --discovery #DISCOVERY_URL
-      ExecStop=-/usr/bin/docker stop $ETCD_NAME
-  - name: etcd2.service
-    mask: true
-  - name: docker-tcp.socket
-    command: start
-    enable: true
-    content: |
-      [Unit]
-      Description=Docker Socket for the API
+    - name: etcd2.service
+      command: start
+    - name: fleet.service
+      command: start
+    - name: docker-tcp.socket
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=Docker Socket for the API
 
-      [Socket]
-      ListenStream=2375
-      BindIPv6Only=both
-      Service=docker.service
-      [Install]
-      WantedBy=sockets.target
-  - name: flanneld.service
-    command: start
-    content: |
-      [Unit]
-      Description=Network fabric for containers
-      Documentation=https://github.com/coreos/flannel
-      Requires=early-docker.service etcd.service
-      After=etcd.service early-docker.service
-      Before=early-docker.target
+        [Socket]
+        ListenStream=2375
+        Service=docker.service
+        BindIPv6Only=both
 
-      [Service]
-      Type=notify
-      Restart=always
-      RestartSec=5
-      Environment="TMPDIR=/var/tmp/"
-      Environment="DOCKER_HOST=unix:///var/run/early-docker.sock"
-      Environment="FLANNEL_VER=0.5.1"
-      LimitNOFILE=40000
-      LimitNPROC=1048576
-      ExecStartPre=/sbin/modprobe ip_tables
-      ExecStartPre=/usr/bin/mkdir -p /run/flannel
-      ExecStartPre=/usr/bin/touch /run/flannel/options.env
-      ExecStartPre=-/usr/bin/etcdctl mk /coreos.com/network/config '{"Network":"10.244.0.0/16", "SubnetLen": 24, "SubnetMin":"10.244.0.0", "Backend": {"Type": "vxlan"}}'
-      ExecStart=/usr/libexec/sdnotify-proxy /run/flannel/sd.sock \
-        /usr/bin/docker run --net=host --privileged=true --rm \
-        --volume=/run/flannel:/run/flannel \
-        --env=NOTIFY_SOCKET=/run/flannel/sd.sock \
-        --env-file=/run/flannel/options.env \
-        --volume=/usr/share/ca-certificates:/etc/ssl/certs:ro \
-        quay.io/coreos/flannel:${FLANNEL_VER} /opt/bin/flanneld --ip-masq=true
+        [Install]
+        WantedBy=sockets.target
+    - name: update-engine.service
+      command: stop
+      enable: false
```

Comment:
We should also have `mask: true` here since we're removing the service which previous had masked it: https://coreos.com/os/docs/latest/cloud-config.html